.github/workflows/gradle.yml

```diff
@@ -0,0 +1,20 @@
+name: Gradle Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build with Gradle
+      run: ./gradlew -Pversion=$GITHUB_REF release --scan --no-daemon -i
+      env:
+        BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
```

Comment:
don't forget to add these secrets (seems to be missing at the moment)

Reply:
Yep, I'll add when this is ready.

Reply:
Done